Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM golang:1.6-onbuild
-MAINTAINER Matt Crane <mcrane@snapbug.geek.nz>
+FROM        prom/busybox:latest
```

Comment:
Done.

On Wed, May 11, 2016 at 2:18 AM, Steve Durrheimer <notifications@github.com>
wrote:

> In Dockerfile
> <https://github.com/prometheus/memcached_exporter/pull/4#discussion_r62796444>
> :
>
> > @@ -1,5 +1,7 @@
> > -FROM golang:1.6-onbuild
> > -MAINTAINER Matt Crane <mcrane@snapbug.geek.nz>
> > +FROM        prom/busybox:latest
>
> Can you change the FROM to quay.io/prometheus/busybox:latest ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/memcached_exporter/pull/4/files/de2b660b0fa1d1229e8f7953b65c3ac0240293e8#r62796444>
>